apps/mark/frontend/src/pages/idle_page.tsx

```diff
@@ -28,7 +27,6 @@ export function IdlePage(): JSX.Element {
   useEffect(() => {
     async function reset() {
       setIsLoading(true);
-      await sleep(1000); // Allow "Clearing ballot" text to display
```

Comment:
This sleep is no longer necessary because `endVoterSession` is now truly async again